src/cmd/linuxkit/builder.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/linuxkit/linuxkit/src/cmd/linuxkit/pkglib"
+	log "github.com/sirupsen/logrus"
+)
+
+func builderUsage() {
+	invoked := filepath.Base(os.Args[0])
+	fmt.Printf("USAGE: %s builder command [options]\n\n", invoked)
+	fmt.Printf("Supported commands are\n")
+	// Please keep these in alphabetical order
+	fmt.Printf("  du\n")
+	fmt.Printf("  prune\n")
+	fmt.Printf("\n")
+	fmt.Printf("'options' are the backend specific options.\n")
+	fmt.Printf("See '%s builder [command] --help' for details.\n\n", invoked)
+}
+
+// Process the builder
+func builder(args []string) {
```

Comment:
this should be named `pkgBuilder`, in line with the other `pkg` subcommand functions.

Probably also should be `pkgBuilderCommands()` and `pkgBuilderUsage()`.

Reply:
Done, thank you